src/main/java/com/salesforce/dynamodbv2/mt/mappers/sharedtable/impl/MtAmazonDynamoDbBySharedTable.java

```diff
@@ -551,6 +591,121 @@ public UpdateItemResult updateItem(UpdateItemRequest updateItemRequest) {
+        }
+    }
+
+    @Override
+    public CreateBackupResult createBackup(CreateBackupRequest createBackupRequest) {
+        if (backupManager.isPresent()) {
+            Preconditions.checkNotNull(createBackupRequest.getBackupName(), "Must pass backup name.");
+            Preconditions.checkArgument(createBackupRequest.getTableName() == null,
+                "Multitenant backups cannot backup individual tables, table name arguments are disallowed.");
+            backupManager.get().createBackup(createBackupRequest);
+
+            ExecutorService executorService = Executors.newFixedThreadPool(mtTables.keySet().size());
+
+            Set<String> snapshottedTables = Sets.newHashSet();
+            List<Future<SnapshotResult>> futures = Lists.newArrayList();
+            Set<String> origMtTables = ImmutableSet.copyOf(mtTables.keySet());
+            for (String tableName : origMtTables) {
+                String snapshottedTable = tableName + "-copy";
+                snapshottedTables.add(snapshottedTable);
+                mtTables.put(snapshottedTable, mtTables.get(tableName));
+                futures.add(executorService.submit(() ->
+                    backupManager.get()
+                        .getMtBackupTableSnapshotter()
+                        .snapshotTableToTarget(
+                            new SnapshotRequest(createBackupRequest.getBackupName(),
+                                tableName,
+                                snapshottedTable,
+                                getAmazonDynamoDb(),
+                                new ProvisionedThroughput(10L, 10L))
+                        )));
+            }
+            List<SnapshotResult> snapshotResults = Lists.newArrayList();
+            try {
+                for (Future<SnapshotResult> future : futures) {
+                    try {
+                        snapshotResults.add(future.get());
+                    } catch (InterruptedException | ExecutionException e) {
+                        throw new MtBackupException("Error snapshotting table", e);
+                    }
+                }
+            } finally {
+                executorService.shutdown();
+            }
+            try {
+                backupManager.get().getMtBackupTableSnapshotter();
+                for (SnapshotResult snapshotResult : snapshotResults) {
+                    backupManager.get().backupPhysicalMtTable(createBackupRequest,
+                        snapshotResult.getTempSnapshotTable());
+                }
+                MtBackupMetadata finishedMetadata = backupManager.get().markBackupComplete(createBackupRequest);
+                return new CreateBackupResult().withBackupDetails(
+                    new BackupDetails()
+                        // TODO: maybe this should be the ARN for the global S3 metadata file for this backup
+                        .withBackupArn(finishedMetadata.getMtBackupName())
```

Comment:
How will ZOS use this ARN?

Reply:
good q. I'm thinking to return these Arn's in listBackup calls, so tenant context specified calls to listBackup's return backups available to said tenant of all virtual tables (if unspecified) with these ARN's encoded within mt-dynamo, available for restore. 

Will post an update to listBackups accordingly.